tests/cypress/support/commands.js

```diff
@@ -24,110 +24,112 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
 
+const getIFrame = () => cy.get('iframe').its('0.contentDocument').should('exist').its('body').should('not.be.undefined').then(cy.wrap)
+
```

Comment:
@shailee-m I am not sure that this would be required. cy.get function would work fine.

Reply:
@Cronus1007  The tests are not running without this. When the browser is loaded cypress cannot find the DOM as it is  loaded from an iframe